tests/test_linters.py

```diff
@@ -19,9 +21,16 @@ def test_pylint(directory):
     _test_linter(PYLINT_COMMAND, directory)
 
 
-def _test_linter(command, path):
+def test_black(directory):
+    _test_linter(BLACK_COMMAND, directory, BLACK_OPTIONS)
+
+
+def _test_linter(command, path, options=None):
+    cmd = [command, path]
+    if options is not None:
+        cmd += options
     try:
-        check_output([command, path])
+        check_output(cmd)
```

Comment:
i my opinion `check_output([command, path, option or ''` will be better and cleaner

Reply:
But won't execute:
`In [6]: command = 'black'
In [7]: path = 'tests'
In [8]: options = None
In [9]: check_output([command, path, options or ''])
Usage: black [OPTIONS] [SRC]...
Try "black -h" for help.

Error: Invalid value for "[SRC]...": Path "" does not exist.
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)`